src/routes/game/components/GameOverDialog.vue

```diff
@@ -24,11 +24,13 @@
       </template>
       <template v-if="currentMatch">
         <p v-if="currentMatch" class="dialog-text" data-cy="match-result-section">
-          Match against {{ gameStore.opponent.username }}
-          <span>: {{ matchIsOver ? 'Finished' : 'In Progress' }}</span>
+          <!-- Match against opponent: finished / in progress -->
+          {{ t('game.dialogs.gameOverDialog.matchAgainst') }} {{ gameStore.opponent.username }}
+          <span>: {{ t(matchIsOver ? 'game.dialogs.gameOverDialog.finished' : 'game.dialogs.gameOverDialog.inProgress') }}</span>
         </p>
         <p v-if="matchIsOver" class="dialog-text" data-cy="match-winner-message">
-          You {{ playerWinsMatch ? 'won' : 'lost' }} your game against {{ gameStore.opponent.username }}
+          <!-- You won/lost your game against  -->
+          {{ t('game.dialogs.gameOverDialog.you') }} {{ t(playerWinsMatch ? 'game.dialogs.gameOverDialog.won' : 'game.dialogs.gameOverDialog.lost') }} {{ t('game.dialogs.gameOverDialog.yourGameAgainst') }} {{ gameStore.opponent.username }}
```

Comment:
In order to simplify this expression let's refactor the text here into a computed prop that puts all this together so that the template code can simply render the computed property rather than having this logic baked into it